reproducibility_project/src/engines/hoomd/project.py

```diff
@@ -20,24 +20,56 @@ def __init__(self):
 
 
 @Project.operation
-@Project.pre(lambda j: j.sp.simulation_engine == "hoomd")
+@Project.pre(lambda j: j.sp.engine == "hoomd")
 def run_hoomd(job):
     """Run a simulation with HOOMD-blue."""
+    import sys
+
+    import foyer
     import hoomd
     import hoomd.md
     from mbuild.formats.gsdwriter import write_gsd
-    from mbuild.formats.hoomd3_simulation import create_hoomd3_forcefield
+    from mbuild.formats.hoomd_forcefield import create_hoomd_forcefield
+
+    sys.path.append(Project().root_directory())
+    from src.molecules.system_builder import construct_system
+
+    # temporary hack until benzene and ethanol are added
+    try:
+        # Ignore the vapor box
+        filled_box, _ = construct_system(job.sp)
+    except AttributeError:
+        return
+
+    if job.sp.forcefield_name == "trappe-ua":
+        ff = foyer.forcefields.load_TRAPPE_UA()
+    elif job.sp.forcefield_name == "benzene_ua":
```

Comment:
This name was also updated in #62 
```suggestion
    elif job.sp.forcefield_name == "benzene-ua":
```